lib/mastodon/emoji_cli.rb

```diff
@@ -96,5 +145,22 @@ def color(green, _yellow, red)
         :red
       end
     end
+
+    def export_file_name(emoji)
+      if options[:shortcode]
```

Comment:
I don't see a benefit for exporting emojis with their "real" names, because then the tar file won't be fit for import, and also the file names are randomized so they carry no meaning.

Reply:
You're right. I thought about that too but was unsure whether it was random or a hash exporting random makes no sense.